lib/ansible/modules/cloud/ovirt/ovirt_event.py

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright: (c) 2019, Ansible Project
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: ovirt_event
+short_description: Create or delete an event in oVirt/RHV
+author: "Chris Keller (@nasx)"
+version_added: "2.8"
+description:
+    - "Create or delete an event in oVirt/RHV."
+options:
+    state:
+        description:
+            - "Should the event be present/absent."
+            - "The wait option must be set to false when state is absent."
+        choices: ['present', 'absent']
+        default: present
+
+    description:
+        description:
+            - "Message for the event."
+            - "Required when state is present."
+
+    severity:
+        description:
+            - "Severity of the event."
+            - "Required when state is present."
+        choices: ['error', 'normal', 'warning']
+        default: normal
```

Comment:
Include type for other parameters as well.

Reply:
Resolved w/ commit 4fbd13e